frontend/src/pages/News/NewsAndStoriesPage.tsx

```diff
@@ -11,60 +11,76 @@ import type { Article } from './ArticleTypes'
 import CheckboxDropdown from './CheckboxDropdown'
 import NewsAndStoriesPreviewCardOutlined from './NewsAndStoriesPreviewCardOutlined'
 
+interface AuthorItem {
+  value: string
+  label: string
+}
+
 export default function NewsAndStoriesPage() {
+  const isMdAndUp = useIsBreakpointAndUp('md')
```

Comment:
can you just move this down so that it's next to isLgUp? just to colocate similar logic